vmdb/app/assets/javascripts/dialog_import_export.js

```diff
@@ -23,45 +23,44 @@ var listenForDialogPostMessages = function() {
 
 var getAndRenderServiceDialogJson = function(importFileUploadId, message) {
   $('.hidden-import-file-upload-id').val(importFileUploadId);
-
   $.getJSON("service_dialog_json?import_file_upload_id=" + importFileUploadId, function(rows_json) {
     var statusFormatter = function(row, cell, value, columnDef, dataContext) {
       var status_img = "<img src=/images/icons/16/" + dataContext.status_icon + ".png >";
-
       return status_img + dataContext.status;
     };
 
     var inputCheckboxFormatter = function(row, cell, value, columnDef, dataContext) {
+      value = dataContext.name;
```

Comment:
I thought it does have local scope, since it's function parameter. I don't get why it's redefined right on the beginning though...